### ISSUE TITLE
Add an accessibility checker

### ISSUE THREAD

**Issue by lchandelier** [ONLYOFFICE/DesktopEditors#2246]
### This issue is unique.

- [x] I have used the [search tool](https://github.com/ONLYOFFICE/DesktopEditors/issues?q=) and did not find an issue describing my idea.

### Your idea.

Could you please add an accessibility checker to verify easily if there is contrast ratio issues, missing alt text on images,missing headings... all common issues that can be found in Word today?

Thanks!

![Image](https://github.com/user-attachments/assets/e174fe37-eff1-4d01-8375-145bca1e6862)

**Comment by ibnpetr** [comment]
Hello 
That sounds interesting. Do you mean using a ready-to-go service?
Could you please share an example of how it is done in other mobile document editing solutions?

**Comment by lchandelier** [comment]
Hello,
If by "ready-to-go service" you mean an automatic tool detecting basic accessibility issues and providing solutions to fix them, then yes :)

Here is a screenshot of the accessibility checker on LibreOffice. I used bad contrasts, no alternative text on image and multiple line breaks to add space.

<img width="1545" height="530" alt="Image" src="https://github.com/user-attachments/assets/6c8384a1-9228-40ec-91a0-8212cb551a6f" />

**Comment by ibnpetr** [comment]
Thank you again for the suggestion! We can’t promise implementation at this stage, 
but we’ll review it and see whether it can be included in future updates!

**Comment by lchandelier** [comment]
You're welcome :) I hope you will be able to implement it, it will be very helpful to create accessible documents for people with disabilities!
